.dockerignore

```diff
@@ -3,8 +3,11 @@ coverage
 dist
 docs
 node_modules
-test
+**/test/**
```

Comment:
Does this also include `fixtures` folders ?

Reply:
Yes it does ! Should we keep theses folders in production ? It doesn't seems to be used in the application to me

Reply:
Okay then it is all good ! No they should not be used in production.